integration/src/main/java/scripts/testscripts/RemoveUser.java

```diff
@@ -59,6 +72,34 @@ protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi owner
         sharedResourceUser.userEmail,
         "The two test users are distinct");
 
+    // Create a group with group email.
+    groupName = "groupremovetest";
+    ApiClient apiClient = new ApiClient();
+
+    assertNotNull(server.samUri);
```

Comment:
Delete 79-80, we typically don't do this.

Lines 75-97 can be simplified. See [here](https://cs.github.com/DataBiosphere/terra-workspace-manager/blob/d74b5da63e8019a2fa63e10177625e68049483dd/integration/src/main/java/scripts/testscripts/EnablePet.java?q=path%3A%2F%5Eintegration%5C%2Fsrc%5C%2Fmain%5C%2Fjava%5C%2Fscripts%5C%2Ftestscripts%2F+sam#L45) for how another integration test uses SAM.

```
    // Create a group
    groupName = "groupremovetest";
    GoogleApi samGoogleApi = SamClientUtils.samGoogleApi(testUser, server);
        SamRetry.retry(
            () ->
                samGoogleApi.postGroup(groupName));
```

You can replace lines 75-97 with above. You can also delete createGroup/deleteGroup(). 2 issues with createGroup(): Usually we use SamRetry. We don't normally catch ApiException, it just adds noise to the test.